apps/sms/views/shared/js/settings.js

```diff
@@ -72,6 +79,15 @@ var Settings = {
     }
   },
 
+  // return resolve promise with sendReadReport status
+  initSendReadReport: function initSendReadReport(value) {
```

Comment:
gasolin asked to set default value in some config file of setting.
Also, if setting doesn't exist yet. It's false. It's in init()  of settings.js